packages/gotop/build.sh

```diff
@@ -14,8 +15,8 @@ termux_step_make_install() {
 	cd $TERMUX_PKG_SRCDIR
 
 	go build -o gotop \
-	    -ldflags "-X main.Version=v${TERMUX_PKG_VERSION} -X main.BuildDate=$(date +%Y%m%dT%H%M%S)" \
+	    -ldflags "-X main.Version=v${_VERSION} -X main.BuildDate=$(date +%Y%m%dT%H%M%S)" \
 	    ./cmd/gotop
 
-	install -Dm700 -t $TERMUX_PREFIX/bin/gotop ./gotop
+	install -Dm777 -t $TERMUX_PREFIX/bin/gotop ./gotop
```

Comment:
`700` is general requirement for executables and directories, `600` for other files. Even if you set something else here, build-package.sh will take care for you to reduce the access mode scope. There no reason for opening access to *group* and *others* (wouldn't be possible anyway).